src/com/lftechnology/java_training/dec31/GuessNumber.java

```diff
@@ -0,0 +1,89 @@
+package com.lftechnology.java_training.dec31;
+
+import java.util.Scanner;
+
+/**
+ * a Guessing a number game application: User will select the guessing value
+ * from the keyboard and application will generate a random no between 1 and
+ * 100. Check if the guess value matched with random no. or not, if matched show
+ * the guessing value, random value and the no. of attempts.
+ * 
+ * @author utsabn34
+ * 
+ */
+public class GuessNumber {
+	private static Scanner input = new Scanner(System.in);
+
+	public static void main(String[] args) {
+		int randomNum = 0;
+		int guessNum = 0;
+		int countAttempt = 0;
+		String continuePlay = "";
+		do {
+			do {
+				/* only generates new random number if user guessed it right */
+				if (randomNum == guessNum) {
+					randomNum = randomNumGen();
+				}
+				System.out.println("Enter your Guessing number from 1-100");
+				guessNum = isInt(input.next());
+
+				/* giving hint to user */
+				if (guessNum < randomNum) {
+					System.out.println("random num is greater than ur guess");
+				} else if (guessNum > randomNum) {
+					System.out.println("random num is less than your guess");
+				} else {
+					System.out.println("");
+				}
+
+				/* count no of attempt user tried */
+				countAttempt++;
+			} while (!(guessNum == randomNum));
+
+			System.out.println("no of attempt: " + countAttempt);
+			System.out.println("Do you want to play again[y/n]:");
```

Comment:
what about i want to play the game again and, does the countAttempt is as per the new game or it just goes on adding the previous attempt. so make your count attempt to 0 after the guess is successful